docs/kbs.yaml

```
@@ -322,25 +322,25 @@ components:
           description: The token issuer URL.
         tee-pubkey:
           type: object
-          $ref: '#/components/schemas/PublicKey'
+          $ref: '#/components/schemas/TeePubKey'
 
-    PublicKey:
+    TeePubKey:
       required:
-        - kty
         - alg
-        - k
+        - k-mod
+        - k-exp
       properties:
-        kty:
-          type: string
-          description: Key type
         alg:
           type: string
           description: Key Algorithm
-        k:
+        k-mod:
+          type: string
+          description: Key modulus
+        k-exp:
           type: string
-          description: Key data
+          description: Key exponent
       description: >-
-        A JSON Web Key (https://www.rfc-editor.org/rfc/rfc7517) formatted object.
+        An RSA Public Key for the TEE
```

The payload follows the JWK format, although we only support RSA keys:

```suggestion
        A JSON Web Key (https://www.rfc-editor.org/rfc/rfc7517) formatted RSA Public Key.
```